src/index.html

```diff
@@ -1,14 +1,77 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Moyo Header</title>
+
+    <link
+     rel="preconnect"
+     href="https://fonts.googleapis.com"
+    >
+
+    <link
+     rel="preconnect"
+     href="https://fonts.gstatic.com"
+     crossorigin="anonymous"
+    >
+
+    <link
+     href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+     rel="stylesheet"
+    >
+
+    <link
+     rel="stylesheet"
+     href="style.css"
+    >
+
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+
+      <a href="#" class="logo-img">
+        <img src="./images/logo.png" alt="Moyo">
+      </a>
+      
+      <nav class="nav">
+        <ul class="nav-list">
+
+          <li class="nav-item">
+            <a href="#" class="nav-link is-active">Apple</a>
+          </li>
+
+          <li class="nav-item">
+            <a href="#" class="nav-link">Samsung</a>
+          </li>
+
+          <li class="nav-item">
+            <a href="#" class="nav-link">Smartphones</a>
+          </li>
```

Comment:
as previous mentor asked
```suggestion
          <li class="nav-item">
            <a href="#samsung" class="nav-link">Samsung</a>
          </li>

          <li class="nav-item">
            <a href="#smartphones" class="nav-link">Smartphones</a>
          </li>
```